oslo/torch/nn/parallel/data_parallel/_fsdp/fully_sharded_data_parallel.py

```diff
@@ -469,15 +472,42 @@ def FullyShardedDataParallel(
     module,
     parallel_context,
     sharding_strategy: Optional[ShardingStrategy] = None,
-    auto_wrap_policy: Optional[Callable] = None,
+    transformer_wrap_layers: Optional[List] = None,
     backward_prefetch: Optional[BackwardPrefetch] = BackwardPrefetch.BACKWARD_PRE,
     mixed_precision: Optional[MixedPrecision] = None,
     cpu_offload: Optional[CPUOffload] = None,
 ):
-    """
-    TODO: Mingu Kang
-    wrapping transformer layer
-    """
+    fsdp_map_for_hf = _FullyShardedDataParallelMappingForHuggingFace()
+    if transformer_wrap_layers is not None:
+        transformer_wrap_layers_intersection = set()
+        transformer_wrap_layers_str = [layer.__name__ for layer in transformer_wrap_layers]
```

Comment:
how about using __qualname__ ?

Reply:
did you check your code with tests?

Reply:
@hyunwoongko  Yes it worked fine.

I tested it now, __name__ and __qualname__ have the same output in this code.

Reply:
@hyunwoongko  I reflected the comments.